testapi.pm

```diff
@@ -396,6 +397,24 @@ sub wait_screen_change(&@) {
     return 0;
 }
 
+=head2 assert_screen_change
+
+  assert_screen_change(CODEREF [,$timeout]);
+
+Run C<CODEREF> with C<wait_screen_change> but C<die> if screen did not change
+within timeout. Look into C<wait_screen_change> for details.
+
+Example:
+
+  assert_screen_change { send_key 'alt-f4' };
+
+=cut
+
+sub assert_screen_change(&@) {
+    ::wait_screen_change(@_) or die 'assert_screen_change failed to detect a screen change';
```

Comment:
I don't know perl, what does `::` do here? I just want to call the other method in the same file and without that it didn't work. Also, deleting the parameter part `(&@)` on both `wait_screen_change` and `assert_screen_change` did not work, ideas?